src/api.ts

```diff
@@ -312,10 +311,10 @@ export const resolvers = {
       return {};
     },
     appVersion: async (parent: any): Promise<string> => {
-      if (process.env.development) {
+      if (process.env.development || process.env.headless_mode === "true") {
         return require("../package.json")?.version;
-      } else if (remote) {
-        return remote.app.getVersion();
+      } else if (require("@electron/remote")?.app) {
```

Comment:
can we move require("@electron/remote") into a variable ? 
 easier on the eyes and doesn't fetch the module multiple times

Reply:
> can we move require("@electron/remote") into a variable ? easier on the eyes and doesn't fetch the module multiple times

This is done on purpose to avoid a crash if electron remote is not initialized (for example, in headless mode)